src/coreclr/jit/lowerarmarch.cpp

```diff
@@ -1775,6 +1775,50 @@ GenTree* Lowering::LowerHWIntrinsic(GenTreeHWIntrinsic* node)
 
             break;
         }
+        case NI_Sve_GatherVectorFirstFaulting:
+        {
+            LIR::Use use;
+            bool     foundUse = BlockRange().TryGetUse(node, &use);
+
+            if (m_ffrTrashed)
```

Comment:
Is this code adding the extra arg that is checked for in hwintrinsiccodegenarm64.cpp which causes the wrFFR ?

Reply:
Hmm, now I'm wondering if this code was left over from a merge.

Reply:
Ok, this was not left over. @kunalspathak added this for this PR.

When looking at hwintrinsiccodegenarm64.cpp, it does seem to be the case.